src/tooltip/css/objects/_honeycomb.tooltip.objects.tooltip.scss

```diff
@@ -0,0 +1,96 @@
+
+/* Base styles for the entire tooltip and default position*/
+.tooltip, [data-tooltip], .tooltip-top {
+  position: relative;
+  cursor: help;
+
+  /* Shared animations and properties */
+  &:before, &:after {
+    position: absolute;
+    visibility: hidden;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0);
+    opacity: 0;
+    -webkit-transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            -webkit-transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    -moz-transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            -moz-transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    transition:
+            opacity 0.2s ease-in-out,
+            visibility 0.2s ease-in-out,
+            transform 0.2s cubic-bezier(0.25, 0.1, 0.25 ,1);
+    -webkit-transform: translate3d(0, 0, 0);
+    -moz-transform:    translate3d(0, 0, 0);
+    transform:         translate3d(0, 0, 0);
```

Comment:
/some/ vendor-prefixing happens automatically during the build - if you just have the one `transform` rule, then a copy of the rule with a `-webkit-` prefix gets added in to the minified CSS automatically (`-moz-` prefixes don't, I'm not sure why)
If you want to keep the `-moz` version feel free to leave as is, but if you want to cut down on the code you'd probably be fine to remove the vendor-prefixed rules